.rubocop.yml

```diff
@@ -1,38 +1,38 @@
+# TODO: remove this file by either moving cops here or fixing code
+inherit_from:
+  - .rubocop_todo.yml
```

Comment:
Is this `inherits` needed to make tests pass? I see there are a few errors locally without it, but copying the TODO directly from Foreman is disabling cops that are not violated here.